images/kubeadm/runner

```diff
@@ -26,7 +26,7 @@ if [ ! -e kubernetes-anywhere ]; then
 
   # Explicitly version this dependency so that upstream commits can't
   # immediately break e2e jobs, and we have control over upgrading/downgrading.
-  git -C kubernetes-anywhere checkout 3ea5e1f14e57c1eb8c21455348d9825f277be350
+  git -C kubernetes-anywhere checkout b5a8c03c3d08b068dc2af6208005b2dd342d1356
```

Comment:
I think we should move cloning this into the job definition, this doesn't need to be in an image runner, that way you won't have to wait for us to push a new image each time.

Reply:
Is this what we should do or what we already done? Sorry I haven't follow this repo for a long time.
Didn't notice we have a flag of `kubernetes-anywhere-version` or something like this, but if we do have, that would be nice. : )

Reply:
well no, we have a more generic flag to checkout repositories with `--repo` on the jobs, we can configure the jobs to fetch this repo instead of baking it into an image runner. 

Similar to how the job fetches kubernetes to test it with kubeadm, we can fetch kubernetes-anywhere as well (we already fetch multiple repos for many jobs, eg kubernetes/kubernetes + kubernetes/release)